Assets/HoloToolkit/Input/Scripts/Utilities/MotionControllerVisualizer.cs

```diff
@@ -38,18 +38,16 @@ public class MotionControllerVisualizer : MonoBehaviour
 
         [Tooltip("This material will be used on the loaded glTF controller model. This does not affect the above overrides.")]
         [SerializeField]
-        protected UnityEngine.Material GLTFMaterial;
+        protected Material GLTFMaterial;
```

Comment:
This will break 2017.2 building when this change is merged back into the dev branch (or when master i updated to work with 2017.2) because there's a Material in the GLTF namespace as well. Likely, renaming it to GLTFMaterial is the best fix.

Reply:
I'll look into it and double check. Thanks for the heads up!